starters/deno-oak-denodb/src/__tests__/resolvers_test.ts

```diff
@@ -0,0 +1,86 @@
+import { getTechnologies, getTechnology } from '../graphql/resolvers/query_handler.ts';
```

Comment:
Is this file name supposed to be `resolvers_test` or `queries_test`?

Reply:
Well these functions under test are resolvers for the queries, so I'd say it should be `resolvers_test` but good question. 🤔

Reply:
We can rename them to `query_handler_test` to be clear that it's testing functions from `query_handler`.